stratum/hal/bin/barefoot/deb/start-stratum.sh

```diff
@@ -50,6 +50,15 @@ if ! grep hugetlbfs /etc/mtab | grep -q /mnt/huge; then
     fi
 fi
 
+# Use the BSP on Wedge switches by default.
+if [[ "$PLATFORM" == "x86-64-accton-wedge100bf-32x-r0" ]] || \
+   [[ "$PLATFORM" == "x86-64-accton-wedge100bf-32qs-r0" ]] || \
+   [[ "$PLATFORM" == "x86-64-accton-wedge100bf-65x-r0" ]]; then
+    BF_SWITCHD_CFG=/usr/share/stratum/tofino_skip_p4.conf
+else
+    BF_SWITCHD_CFG=/usr/share/stratum/tofino_skip_p4_no_bsp.conf
+fi
+
 # Set up port map for device
 PORT_MAP="/etc/stratum/$PLATFORM/port_map.json"
 if [ ! -f "$PORT_MAP" ]; then
```

Comment:
We can skip this step if we are using the BSP. But it also doesn't hurt to have it.

Reply:
Don't we still need the map if a user overrides the `bf_switchd_cfg` flag to non-bsp?